en/editor/console/index.md

```diff
@@ -0,0 +1,28 @@
+# Console
+
+![main](index/main.jpg)
+
+**Console** display errors, warnings, or other log information generated by the editor and engine. Information of different important levels is displayed in different colors.
+
+## Abourt log
+The log system is based on `js` **console** model. When the editor is running, will captures the necessary logs printed out of the **console** in each process and distinguishes them by color.
+
+  - **Log** (__console.log__): <font color=#b6b6b6>This is gray text</font>, editor and developers think the content of need printing to **console**.
+  - **Warn** (__console.warn__): <font color=#ebbe09>This is yellow text</font>, when the program is running, it encounter abnormal situation that does not affect  the result.
```

Comment:
it encounter abnormal situation that does not affect  the result.
--> abnormal situations that do not affect the program's result are encountered and outputted.